.github/workflows/webassembly.yml

```diff
@@ -0,0 +1,80 @@
+name: Webassembly
+on:
+  schedule:
+    # at 00:00 on Wednesday and Saturday
+    - cron: '0 0 * * 3,6'
```

Comment:
Should there be a schedule?

Reply:
no, copy-paste from another workflow.

removed